mne/io/persyst/persyst.py

```diff
@@ -108,8 +111,10 @@ def __init__(self, fname, preload=False, verbose=None):
             # Patient (All optional)
             elif section == 'patient':
                 patient_dict[key] = val
+            # Comments (turned into mne.Annotations)
             elif section == 'comments':
-                comments_dict[key] = val
+                comments_dict[key].append(val)
```

Comment:
then this can just be the following if you still want a one-liner
```suggestion
                comments_dict[key] = comments_dict.get(key, list()) + [val]
```
or split it into two lines if you want